src/inverseRobot.js

```diff
@@ -8,6 +8,19 @@
 
 function inverseRobot(robot) {
   // write code here
+  const trueRobot = {};
+
+  for (const key in robot) {
+    const paramName = robot[key];
+
+    if (robot[key] in trueRobot) {
```

Comment:
don't forget to use your variable here

Reply:
😢

Reply:
I'm sorry, don't understand what I should do here, I changed the name for my variable everywhere, how i can use this variable here?

Reply:
Understood